llarp/config/config.cpp

```diff
@@ -834,48 +834,66 @@ namespace llarp
   }
 
   LinksConfig::LinkInfo
-  LinksConfig::LinkInfoFromINIValues(std::string_view name, std::string_view value)
+  LinksConfig::LinkInfoFromINIValues(
+      std::string_view name, std::string_view value, const net::Platform& net)
   {
     // we treat the INI k:v pair as:
-    // k: interface name, * indicating outbound
+    // k: interface name, * indicating outbound, inbound for inbound addr, outbound for outbound
```

Comment:
This was already kinda weird for an ini file, but I think is weird enough to merit a set of examples.

Reply:
I don't think it needs an example: the k:v pair stuff is what is being deprecated here; the (non-deprecated) config is now:
```
[bind]
inbound=1.2.3.4:5678
outbound=5.6.7.8:9876
```